### PR TITLE
V6/update home features

### PR DESCRIPTION
**Pull request by hsurf22**
### Devop

* \[x] Updated CHANGELOG.md
* \[x] Add PR label
   - Replace features images to actual working components

https://user-images.githubusercontent.com/9893774/112705394-cbf60c00-8e5b-11eb-8bcd-8ddc4892dbe4.mp4

### CODE REVIEW

File: src/views/components-default/HomeFeaturesSend.vue [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  components: {},
+  data: () => ({
+    data: 0
+  }),
+  computed: {
+    ...mapState('wallet', ['balance', 'web3', 'address']),
+    ...mapState('global', ['online']),
+    ...mapState('external', ['ETHUSDValue']),
+    ...mapGetters('global', ['network', 'gasPrice']),
+    ...mapGetters('wallet', ['balanceInETH', 'tokensList']),
```
Comment (olgakup) [inline src/views/components-default/HomeFeaturesSend.vue:51]: what are this getters for in here?

Comment (hsurf22) [reply src/views/components-default/HomeFeaturesSend.vue:51]: computed: {
    ...mapState('wallet', ['balance', 'web3', 'address']),
    ...mapState('global', ['online']),
    ...mapState('external', ['ETHUSDValue']),
    ...mapGetters('global', ['network', 'gasPrice']),
    ...mapGetters('wallet', ['balanceInETH', 'tokensList']),
    tokens() {
      const eth = {
        name: this.network.type.name,
        symbol: this.network.type.name,
        subtext: this.network.type.name_long,
        value: this.network.type.name_long,
        balance: this.balance,
        img: this.network.type.icon,
        decimals: 18,
        market_cap: null,
        price_change_24h: null
      };

      const copiedTokens = this.tokensList.slice();
      copiedTokens.unshift(eth);
      return copiedTokens;
    }
  }


<mew-select label="Type" :items="tokens" />

**Comment by olgakup** [comment]
Can you remove repeating dapps please
Screenshot from 2021-03-30 11-41-49

**Comment by hsurf22** [comment]
Screenshot from 2021-03-30 17-43-50

**Comment by hsurf22** [comment]
Screenshot from 2021-03-30 18-20-10

**Review by olgakup — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -34,12 +34,11 @@
     "@coolwallets/transport-web-ble": "0.1.0",
     "@coolwallets/wallet": "0.2.0-beta.1",
     "@ensdomains/dnsregistrar": "0.4.0",
-    "@myetherwallet/mew-components": "0.5.46-beta",
-    "@myetherwallet/mewconnect-web-client": "2.1.23-beta.8",
```
Comment (olgakup) [inline package.json:38]: it removed it from here

**Pull request merged**